source/components/the-component-lifecycle.md

```diff
@@ -1,25 +1,137 @@
-Part of what makes components so useful is that they let you take
-complete control of a section of the DOM.  This allows for direct DOM
-manipulation, listening and responding to browser events, and using 3rd
-party JavaScript libraries in your Ember app.
+Part of what makes components so useful is that they let you take complete control of a section of the DOM.
+This allows for direct DOM manipulation, listening and responding to browser events, and using 3rd party JavaScript libraries in your Ember app.
 
-As components are rendered, re-rendered and finally removed, Ember
-provides _lifecycle hooks_ that allow you to run code at specific
-times in a component's life.
+As components are rendered, re-rendered and finally removed, Ember provides _lifecycle hooks_ that allow you to run code at specific times in a component's life.
 
-To get the most use out of a component, it is important to understand
-these lifecycle methods. The following hooks are a few of the most
-useful and commonly used in Ember apps.
+To get the most use out of a component, it is important to understand these lifecycle methods.
 
-## Attaching to the Component Element
+To make use of these hooks simply override them within your component.
```

Comment:
I guess the thing that might be confusing is that most of the lifecycle hooks aren't actually functions on the parent class.  It seems to be more of a template pattern where the rendering code will call these methods if they are there.

So a developer might think (as I initially did) that since I'm not overriding any parent functionality, then I don't have to call `_super`, but there still is a good case to use it because of possible mixins to your components which I mention here.